test/components/notice_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'component_test_helper'
+
+class NoticeTest < ComponentTestCase
+  def component_name
+    "notice"
+  end
+
+  test "fails to render when no parameters are given" do
```

Comment:
Not sure that's possible - the integration test couldn't test that the component errors if a required parameter isn't passed.